dev/io.openliberty.microprofile.telemetry.2.0.internal/src/io/openliberty/microprofile/telemetry20/internal/http/MPTelemetryHTTPMetricsAdapterImpl.java

```diff
@@ -54,6 +56,7 @@ public class MPTelemetryHTTPMetricsAdapterImpl implements HTTPMetricAdapter {
     private static final List<Double> BUCKET_BOUNDARIES_LIST = Arrays.asList(BUCKET_BOUNDARIES);
 
     private final WeakHashMap<OpenTelemetry, DoubleHistogram> httpHistogramMap = new WeakHashMap<OpenTelemetry, DoubleHistogram>();
+    private final ReadWriteLock rwl = new ReentrantReadWriteLock();
```

Comment:
Please add a comment that access to `httpHistogramMap` must be synchronized with the `ReadWriteLock`.

Please also rename `rwl` to `httpHistogramMapLock` to indicate what it's for.